packages/apps/graph/src/index.ts

```diff
@@ -30,12 +33,43 @@ if (dotenv.NODE_ENV === 'development') {
 
 const schema = builder.toSchema();
 
-const plugins = [extensionsPlugin()];
+const plugins: Plugin[] = [extensionsPlugin()];
 
 if (dotenv.COMPLEXITY_EXPOSED) {
   plugins.push(complexityPlugin(schema));
 }
 
+const tracingPlugin: Plugin = {
+  onExecute: ({ setExecuteFn, executeFn }) => {
+    setExecuteFn((options) =>
+      Sentry.startSpan(
+        {
+          op: 'graphql.execute',
+          name: options.operationName ?? '<unnamed operation>',
+          forceTransaction: true,
+          attributes: {
+            [AttributeNames.OPERATION_NAME]: options.operationName ?? undefined,
+            // [AttributeNames.SOURCE]: print(options.document),
+          },
+        },
+        () => executeFn(options),
+      ),
+    );
+  },
+};
+
+if (dotenv.SENTRY_DSN) {
+  console.log(
+    ` ✔ starting with sentry ${dotenv.NODE_ENV}-${dotenv.ENVIRONMENT}`,
+  );
+  Sentry.init({
+    dsn: dotenv.SENTRY_DSN,
+    tracesSampleRate: 1,
+    environment: dotenv.ENVIRONMENT,
```

Comment:
Should this be `${dotenv.NODE_ENV}-${dotenv.ENVIRONMENT}`?

```suggestion
    environment: `${dotenv.NODE_ENV}-${dotenv.ENVIRONMENT}`,
```

I think just mainnet/testnet isn't enough to differentiate. Especially when we run it in development too.

Would be nice to query the configured chainwebapi (https://api.chainweb.com/info) and use the `nodeversion`

Reply:
I thought about this. The default value for NODE_ENV is `production`. If a developer incorrectly sets environment variables (or just forgets them), you could end up with `production-local` which sounds confusing.

Having said that, the default for ENVIRONMENT is `mainnet`. If a developer sets the SENTRY_SDN but forgets the environment, they could pollute the logging...